viadot/tasks/genesys.py

```diff
@@ -449,3 +460,38 @@ def run(
             logger.info("Downloaded the data from the Genesys into the CSV.")
 
             return [file_name]
+
+        elif view_type is None and end_point == "conversations":
+            data_list = []
+
+            for Id in conversationId_list:
+                json_file = genesys.genesys_api_connection(
+                    post_data_list=post_data_list,
+                    end_point=f"{end_point}/{Id}",
+                    method="GET",
+                )
+                logger.info(f"Generated webmsg_response for {Id}")
+
+                attributes = json_file["participants"][0]["attributes"]
+                temp_dict = {
+                    key: value for (key, value) in attributes.items() if key in key_list
+                }
+                temp_dict["conversationId"] = json_file["id"]
+                data_list.append(temp_dict)
+
+            df = pd.DataFrame(data_list)
+            df = df[df.columns[-1:]].join(df[df.columns[:-1]])
+
+            start = start_date.replace("-", "")
+            end = end_date.replace("-", "")
+
+            file_name = f"WebMessage_{start}-{end}".upper() + ".csv"
```

Comment:
Do you need the `upper` function? You enter the same text each time, so you can use capital letters in the text.